autoload/go/guru.vim

```diff
@@ -111,7 +111,8 @@ function! s:sync_guru(args) abort
 
   if !has_key(a:args, 'disable_progress')
     if a:args.needs_scope
-      call go#util#EchoProgress("analysing with scope ". result.scope . " ...")
+      call go#util#EchoProgress("analysing with scope ". result.scope .
```

Comment:
analysing -> analyzing

Reply:
Both are correct I think. The Vim spell checks seems to prefer the "s" variant.

Reply:
Interesting! I looked it up and discovered that `analysing` is British english, while `analyzing` is American English: http://writingexplained.org/analyse-or-analyze-difference.

I've been considering making a pass through docs and help messages and correcting grammar and spelling. I would likely default to American English; do you think vim-go should be using British English?

Reply:
Let's go with American English. I try to use American English whenever I can.

Reply:
I don't really care; no matter how you spell it, everyone will know what you mean anyway ;-)

Note that I didn't write this line btw, I just added the notice about `:help` here. The `analys*` spelling is used consistently throughout the code base at the moment, including some variables. Seems a bit out of scope to change all of that here (but feel free to change it in another PR).